vassal-app/src/main/java/VASSAL/build/module/properties/PropertyChangerConfigurer.java

```diff
@@ -303,4 +314,34 @@ public void setValue(String s) {
 
   public interface Constraints extends PropertyPrompt.Constraints, IncrementProperty.Constraints, PropertySource {
   }
+
+  @Override
+  public void setHighlighted(boolean highlighted) {
+    super.setHighlighted(highlighted);
+    typeConfig.setHighlighted(highlighted);
+    incrConfig.setHighlighted(highlighted);
+    valueConfig.setHighlighted(highlighted);
+    promptConfig.setHighlighted(highlighted);
+    validValuesConfig.setHighlighted(highlighted);
+  }
+
```

Comment:
Could you do something like
```
final List<Configurer> configurers = List.of(typeConfig, incrConfig, valueConfig, promptConfig, validValuesConfig);

@Override
public void setHighlighted(boolean highlighted) {
  super.setHighlighted(highlighted);
  configurers.forEach(c -> c.setHighlighted(highlighted));
}
```
and similarly for the two functions following?

Reply:
Done